src/main/java/uk/gov/hmcts/reform/civil/service/flowstate/FlowStateAllowedEventService.java

```diff
@@ -426,18 +409,15 @@ public class FlowStateAllowedEventService {
                 DISMISS_CLAIM,
                 ADD_CASE_NOTE,
                 INITIATE_GENERAL_APPLICATION,
-                CREATE_SDO,
-                NotSuitable_SDO
+                CREATE_SDO
             )
         ),
         entry(
             PAST_CLAIM_DETAILS_NOTIFICATION_DEADLINE_AWAITING_CAMUNDA.fullName(),
             List.of(
                 DISMISS_CLAIM,
                 ADD_CASE_NOTE,
-                INITIATE_GENERAL_APPLICATION,
-                CREATE_SDO,
```

Comment:
why did you remove CREATE_SDO here

Reply:
To align with a unit test when I was rebasing - should it be there? I can put it back & adjust unit test.